zubhub_backend/zubhub/creators/tests/test_views.py

```diff
@@ -0,0 +1,58 @@
+from rest_framework.test import APITestCase
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework.test import APIClient
+from rest_framework import status
+from creators.models import Creator, Location
+
+
+creator = get_user_model()
+
+
+class FetchAuthUserApiTests(TestCase):
+    """Test the users API"""
+
+    def setUp(self):
+        self.client = APIClient()
+        self.location = Location.objects.create(name="Nigeria")
+        self.payload = {
+            "username": "test_dummy",
+            "email": "test_dummy@gmail.com",
+            "password1": "qwertyu12345",
+            "password2": "qwertyu12345",
+            "phone": "+57778899066",
+            "dateOfBirth": "1998-02-20",
+            "location": "Nigeria",
+            "bio": "ddd",
+            "subscribe": True,
+        }
+
+    def test_create_valid_user_success(self):
+        """Test creating user with valid payload is successful"""
+        payload_false = {"username": "bean", "password": "boom12345"}
+        res = self.client.post(reverse("creators:signup_creator"), self.payload)
+        res_false = self.client.post(reverse("creators:signup_creator"), payload_false)
```

Comment:
We can remove this line from test now as this is covered in the unsuccessful scenario test case.